src/widgets/RTSPtoWeb/VideoFeed.tsx

```diff
@@ -0,0 +1,41 @@
+import { useRef, useEffect, useState } from 'react';
+import videojs from 'video.js';
+import 'video.js/dist/video-js.css';
+
+const VideoFeed: React.FC<VideoFeedProps> = ({ src }) => {
+  const videoRef = useRef(null);
+  const [player, setPlayer] = useState<ReturnType<typeof videojs>>();
+
+  useEffect(() => {
+    // make sure Video.js player is only initialized once
+    if (!player) {
+      const videoElement = videoRef.current;
+      if (!videoElement) return;
+
+      setPlayer(videojs(videoElement, {}, () => {
+          // console.log("player is ready");
+        })
+      );
+    }
+  }, [videoRef]);
+
+  useEffect(() => () => {
+      if (player) {
+        player.dispose();
+      }
+    }, [player]);
+
+  return (
+    <div>
+      <video className="video-js" ref={videoRef} controls>
+        <source src={src} type="application/x-mpegURL" />
+      </video>
+    </div>
+  );
+};
+
+interface VideoFeedProps {
+  src: string;
```

Comment:
Abbreviations are bad practice. I prefer them to be spelled out. Consider using ``source``.

Reply:
> Abbreviations are bad practice. I prefer them to be spelled out. Consider using `source`.

Hi I need some help, can you help me?
I come from Python world.

Reply:
Just use a different name for the type and adjust all usages